diff --git a/configure.ac b/configure.ac
index 085ceab..2e120c1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -99,10 +99,10 @@ if test "$enable_ssh" != "no"; then
   LIBS=$old_LIBS
 fi
 
-# pam
-AC_CHECK_HEADER([security/pam_appl.h], ,
-  [AC_MSG_ERROR([Couldn't find PAM headers. Try installing pam-devel])]
-)
+# # pam
+# AC_CHECK_HEADER([security/pam_appl.h], ,
+#   [AC_MSG_ERROR([Couldn't find PAM headers. Try installing pam-devel])]
+# )
 PAM_LIBS="-lpam"
 COCKPIT_SESSION_LIBS="$COCKPIT_SESSION_LIBS $PAM_LIBS"
 
