diff --git a/src/tls/cockpit-certificate-ensure.c b/src/tls/cockpit-certificate-ensure.c
index f553c2a1f..e9601a6f8 100644
--- a/src/tls/cockpit-certificate-ensure.c
+++ b/src/tls/cockpit-certificate-ensure.c
@@ -43,7 +43,7 @@
 
 // Cross-reference with cockpit-certificate-helper.in
 #define COCKPIT_SELFSIGNED_FILENAME  "/cockpit/ws-certs.d/0-self-signed.cert"
-#define COCKPIT_SELFSIGNED_PATH      PACKAGE_SYSCONF_DIR COCKPIT_SELFSIGNED_FILENAME
+#define COCKPIT_SELFSIGNED_PATH      "/etc" COCKPIT_SELFSIGNED_FILENAME
 
 // Renew certificates with less than 30 days validity
 #define EXPIRY_THRESHOLD (30 * 24 * 60 * 60)
